### ISSUE TITLE
yourls replacing all &'s in link with &amp;

### ISSUE THREAD

**Issue by ozh** [YOURLS/YOURLS#114]
This is a **COPY** of **Issue 114: [yourls replacing all &'s in link with &amp;](http://code.google.com/p/yourls/issues/detail?id=114)**, filed on Google Code before the project was [moved on Github](https://github.com/ozh/google-issues-to-github).
- Submitted on 2009-10-03T01:57:46.000Z by [lloyd.ha...@gmail.com](http://code.google.com/u/116030943151622272800/)
- Status: Invalid

Please **review** the original issue and especially its comments. **Comments here on closed issues will be ignored**. Thanks.
## Original description

```
I'm trying to use yourls to shorten some long links with many &amp;, but they
are being replaced with &amp;amp; due to the use of mysql_real_escape_string

I added in a simple hack to do this, but instead of having to re do it each
time I upgrade is this something that could be added into the core?

Basically I just added the second line:

    $url = mysql_real_escape_string(yourls_sanitize_url($url));
    $url = str_replace("&amp;amp;","&amp;",$url);

Works, but not sure if it's the best method :)
```
